integration-tests/src/test/java/com/redhat/service/bridge/integration/tests/common/BridgeUtils.java

```diff
@@ -0,0 +1,48 @@
+package com.redhat.service.bridge.integration.tests.common;
+
+import org.keycloak.representations.AccessTokenResponse;
+
+import io.restassured.http.ContentType;
+import io.restassured.specification.RequestSpecification;
+
+import static io.restassured.RestAssured.given;
+
+public class BridgeUtils {
+    protected static final String USER_NAME = getSystemProperty("bridge.token.username");
+    protected static final String PASSWORD = getSystemProperty("bridge.token.password");
+    protected static final String CLIENT_ID = getSystemProperty("bridge.client.id");
+    protected static final String CLIENT_SECRET = getSystemProperty("bridge.client.secret");
+
+    protected static String token;
+    protected static String keycloakURL = getSystemProperty("key-cloak.url");
+
+    public static RequestSpecification jsonRequestWithAuth() {
+        if (token == null) {
+            token = getAccessToken();
```

Comment:
Please add a possibility to supply access token directly.
In PR check the access token will be retrieved using internal Keycloak URL which is not available externally.
Access token could be passed using env variable.